queue_services/entity-emailer/src/entity_emailer/email_processors/incorp_notification.py

```diff
@@ -12,13 +12,166 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 """Email processing rules and actions for completeing incorporation."""
+import base64
+from datetime import datetime
+from http import HTTPStatus
+from pathlib import Path
+
+import requests
 from entity_queue_common.service_utils import logger
+from flask import current_app
+from jinja2 import Environment, FileSystemLoader, Template
+from legal_api.models import Filing
+from legal_api.utils.legislation_datetime import LegislationDatetime
+from sentry_sdk import capture_message
+
+from entity_emailer.email_processors import substitute_template_parts
+
+
+ENV = Environment(loader=FileSystemLoader('email-templates'), autoescape=True)
+
+
+def _get_pdfs(stage: str, token: str, business: dict, filing: Filing, filing_date_time: str):
+    """Get the pdfs for the incorporation output."""
+    pdfs = []
+    headers = {
+        'Accept': 'application/pdf',
+        'Authorization': f'Bearer {token}'
+    }
+    if stage == 'filed':
+        # IA pdf
+        inc_app = requests.get(
+            f'{current_app.config.get("LEGAL_API_URL")}/businesses/{business["identifier"]}/filings/{filing.id}',
+            headers=headers
+        )
+        if inc_app.status_code != HTTPStatus.CREATED:
```

Comment:
I think 201 is when something gets created. It is ok for receipts as it is a post request. Just wondering whether it should be HTTPStatus.OK here or why our API returns 201 for a GET?

Reply:
oops! will fix